### PR TITLE
adding total section to the stats docs

### PR DESCRIPTION
**Pull request by traetox**

### CODE REVIEW

File: search/stats/stats.md [hunk 1]
```diff
@@ -114,4 +115,20 @@ stats mean(Bytes) stddev(Bytes) by SrcIP over 5m
 
 When sent to the chart module, the results will be calculated over a 5 minute window rather than the standard 1 second.
 
-Note: Only one time window can be specified, and the time window is applied to all operations.  The time window must also be the LAST argument to stats
+Note: Only one time window can be specified, and the time window is applied to all operations.  The time window must also be the LAST argument to statsa
```
Comment (floren) [inline search/stats/stats.md:118]: typo

File: search/stats/stats.md [hunk 2]
```diff
@@ -114,4 +115,20 @@ stats mean(Bytes) stddev(Bytes) by SrcIP over 5m
...
+
+## sum() vs total()
+
+The `sum` and `total` operators behave exactly the same in every context except chart.  When using the chart renderer the total module will not "reset" values across time buckets.
+
+The behavior difference is best demonstrated in a screenshot generated using the following query:
+
+```
+tag=zeekconn fields -d "\t" [8] as dur |
```
Comment (floren) [inline search/stats/stats.md:127]: put the pipe on the next line instead

File: search/stats/stats.md (hunk 1 above)
Comment (traetox) [reply search/stats/stats.md]: fixed

File: search/stats/stats.md (hunk 2 above)
Comment (traetox) [reply search/stats/stats.md]: fixed

**Pull request merged**